x-pack/plugins/maps/public/routing/state_syncing/global_sync.ts

```diff
@@ -18,18 +18,15 @@ export function getGlobalState() {
   return kbnUrlStateStorage.get('_g');
 }
 
-export function updateGlobalState(newState: unknown) {
+export function updateGlobalState(newState: unknown, skipFlush = true) {
```

Comment:
Seems odd that method accepts `skipFlush` but then is checking for the opposite. How about making parameter `flushUrlState` then the check is not need to be negated and better shows how the parameter effects the methods behavior